specs/engage-digital_openapi3.yaml

```
@@ -5146,6 +5146,179 @@ paths:
+            is 150.
+          in: query
+          name: limit
+          required: false
+          schema:
+            format: int32
+            type: integer
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+               $ref: '#/components/schemas/GetAllUserCapacities'
+          description: Success
+      summary: Getting all User Capacities
+      tags:
+        - UserCapacity
+    post:
+        description: >-
+          This method creates a new user capacity. In case of success it renders the
+          updated record, otherwise, it renders an error (422 HTTP code).
+          Please note that the order of the parameters is important, as the default and
+          max capacities will apply to the immediately above channel_id.
+
+          Authorization​: only users who can manage topology.
+        operationId: createUserCapacity
+        parameters:
+          - description: User Capacity name.
+            in: query
+            name: name
+            required: true
+            schema:
+              type: string
+          - description: The id of the channel on which the capacities applies.
+            in: query
+            name: channels[][channel_id]
+            required: false
+            schema:
+              type: string
+          - description: the default capacity to apply on the previous channel id.
+            in: query
+            name: channels[][default_capacity]
+            required: false
+            schema:
+              type: integer
+          - description: the maximum capacity to apply on the previous channel id.
+            in: query
+            name: channels[][max_capacity]
+            required: false
+            schema:
+              type: integer
+        responses:
+          '200':
+            content:
+              application/json:
+                schema:
+                  $ref: '#/components/schemas/UserCapacity'
+            description: Success
+        summary: Creating a user capacity
+        tags:
+          - UserCapacity
+  '/user_capacities/{userCapacityId}':
+    delete:
+      description: >-
+        This method destroys an existing user capacity. It renders the user capacity
+        itself. It renders a 404 if id is invalid or already destroyed.
+
+        Authorization​: only users who can manage topology.
+      operationId: deleteUserCapacity
+      parameters:
+        - in: path
+          name: userCapacityId
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                  $ref: '#/components/schemas/UserCapacity'
+          description: Success
+      summary: Deleting a user capacity
+      tags:
+        - UserCapacity
+    get:
+      description: >-
+        This method renders a user capacity from given id.

```

```suggestion
        This method renders the user capacity corresponding to the id given in parameter.
```

We should also mention that it renders a 404 if the record can't be found